packages/tulips.yml

```
@@ -0,0 +1,5 @@
+section: domain specific libraries
```

```suggestion
name: TULIPS
repo: https://bitbucket.org/elaplace/tulips/src/master/
section: domain specific libraries
```
This should be the correct content now.